docs/concepts/isis-fundamentals/label-dictionary.md

```diff
@@ -0,0 +1,261 @@
+EndGroup
+```
+
+## BandBin Group
+
+This group is used to stored information regarding each band in the cube. This implies that every keyword in the group be an array with a dimension equal to the number of bands in the cube. This group is not required nor if it exists are any particular keywords required. That is, the names of keywords are irrelevant although some instrument- specific programs may expect certain keywords to be available (e.g., OriginalBand). The keywords in this group will be automatically pruned if any program uses the input cube attribute option (e.g., from=input.cub+1,3-4).
+
+```
+Group = BandBin
+  OriginalBand = (1, 2, 3, 4, 5)
+  Center       = (6.78, 6.78, 7.93, 8.56, 9.35) <microns>
+  Width        = (1.01, 1.01, 1.09, 1.16, 1.20) <microns>
+  FilterNumber = (1, 2, 3, 4, 5)
+EndGroup
+```
+
+|Keyword|Type|Description|
+|--- |--- |--- |
+|OriginalBand|Integer|This keyword indicates the original band number at the current band location.  Hence, extracting bands 2 and 5 from a cube via the input cube attribute option would yield OriginalBand = (2,5)|
+|Center, Width|Double|Often these keywords are used to describe the center wavelength and width of each band.|
+
+## ImportLabel Object
+```
+Object = ImportLabel
+  Lblsize  = 536
+  Format   = HALF
+  Type     = IMAGE
+  Bufsiz   = 24576
+  Dim      = 3
+  Eol      = 1
+  Recsize  = 536
+  Org      = BSQ
+  Nl       = 256
+  Ns       = 256
+  Nb       = 1
+  N1       = 256
+  N2       = 256
+  N3       = 1
+  N4       = 0
+  Nbb      = 24
+  Nlb      = 1
+  Host     = SUN-SOLR
+  Intfmt   = HIGH
+EndObject
+```
+
+## Instrument Group 
+
+This group stores specific information regarding the mission and instrument. This information is typically used to perform radiometric calibration as well as in camera models in order to accurately compute ground positions in a raw camera cube. Obviously, the keywords in the group will vary from instrument to instrument. However, there are three required keywords (Name, Spacecraft, and Target). Definitions for Instrument specific keywords can be found in the appropriate documentation for that instrument.
+
+```
+Group = Instrument
+  SpacecraftName  = "Galileo Orbiter"
+  InstrumentId    = "Solid State Imaging System"
+  TargetName      = Io
+  StartTime       = "1996-06-25T17:36:56.858"
+  FrameModeId     = Full
+EndGroup
+```
+
+|Keyword|Type|Description|
+|--- |--- |--- |
+|SpacecraftName|String|Name of the mission/spacecraft|
+|InstrumentId|String|The name of the instrument|
+|TargetName|String|The name of target/body which the instrument is viewing|
+
+## Kernels Group
+
+See [The ISIS Attached Spice Format](../isis-fundamentals/spice-format.md#the-kernels-group).
+
+## Mapping Group
+
+The Mapping Group is used to define map projections. Recommending reading includes the ISIS Map Projection Guide and Map Projections – A Working Manual, Snyder, J.P., U.S. Geological Survey, Bulletin 1532.
```

Comment:
Can we turn this into a link to the publication? https://pubs.usgs.gov/publication/pp1395